tardis/plasma/properties/atomic.py

```diff
@@ -58,18 +59,19 @@ class Lines(BaseAtomicDataProperty):
     outputs = ('lines', 'nu', 'f_lu', 'wavelength_cm')
 
     def _filter_atomic_property(self, lines, selected_atoms):
-        return lines[lines.atomic_number.isin(selected_atoms)]
+        # return lines[lines.atomic_number.isin(selected_atoms)]
+        return lines
 
     def _set_index(self, lines):
 # Filtering process re-arranges the index. This re-orders it.
 # It seems to be important that the lines stay indexed in the correct order
 # so that the tau_sobolevs values are in the right order for the montecarlo
 # code, or it returns the wrong answer.
```

Comment:
My reasoning behind not removing it was that someday in the future when all the filtering and atom data preparation gets implemented in the plasma part having the old functionality at hand might be useful.

But your comment reminded me that BasicAtomDataProperty definitely needs a change.